torch/ao/nn/sparse/quantized/linear.py

```diff
@@ -82,7 +83,7 @@ class Linear(torch.nn.Module):
     A quantized sparse linear module with quantized tensor as inputs and outputs.
     """
     _version = 1
-    _FLOAT_MODULE = torch.nn.Linear
+    _FLOAT_MODULE = (torch.nn.Linear, SparseLinear)
```

Comment:
I am assuming that nn.Linear is needed for the PTQ flow and SparseLinear for the QAT flow. Is that correct?

Reply:
Lets remove the torch.nn.Linear as a supported option, In both flows, we will need to create a SparseLinearModule. In the Post training sparsity case, one could do:
```python
m = nn.Linear(3,5)
# Sparsifier.prepare would essentially do this:
m_sparse = torch.nn.sparse.linear.from_dense(m)
# m_sparse now has the same weights, but the mask is set to identity
m_sparse.set_mask(mask, pattern) # Function to override the mask and define the sparsity pattern
# Now we can convert only the sparse linear module.
```